cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -95,9 +95,10 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             cy.window().then(win => {
               const media = getBlockData('video', win.SIMORGH_DATA.pageData);
               if (media && media.type === 'video') {
-                cy.get(
-                  'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
-                )
+                cy.get('div[class^="StyledVideoContainer"]')
+                  .within(() => {
+                    cy.get('button');
+                  })
```

Comment:
Why is this being changed?

Reply:
With the inclusion of guidance the positioning of the button within the div changes, making the previous test fail

Reply:
Does this not duplicate work being also done in https://github.com/bbc/simorgh/pull/4525?

Reply:
The intention was to modify the work done here #4242 but i can see how it can be a duplication of #4525
Currently looking for an alternative implementaion